docs/api.md

```diff
@@ -0,0 +1,171 @@
+# AUACM API Documentation
+
+The Auburn ACM website uses a RESTful api for managing blog posts, problems,
+submissions, etc. This document is intended to serve as a one stop shop for the
+method arguments and return values.
+
+### Table of Contents
+
+1. Problem Management
```

Comment:
Just for convenience, in the table of contents, you can do something like `[Problem Management](#problem-management)`, which will link to the corresponding section of the doc.